libs/backend/src/scan/cast_vote_records/cast_vote_record_exporter.ts

```diff
@@ -0,0 +1,415 @@
+/* istanbul ignore file */
```

Comment:
Going to add tests in subsequent PRs

Issue to track: https://github.com/votingworks/vxsuite/issues/3837